src/components/dropdown.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference path="../../typings/tsd.d.ts" />
+import {Component, View, NgClass} from 'angular2/angular2';
+
+@Component({
+  selector: 'uib-dropdown',
+  properties: [
+    'open'
+  ]
+})
+@View({
+  directives: [NgClass],
+  template: `<div class="dropdown" [ng-class]="{open: open}">
```

Comment:
Err, I had that suggestion wrong - [here](http://plnkr.co/edit/duj7JGoglJfX4b5oHK6g?p=preview) is a working implementation with changes though. This seems to have CSS ramifications though.